spiketoolkit/postprocessing/waveform_features_library.py

```diff
@@ -0,0 +1,253 @@
+"""
+Functions based on
+https://github.com/AllenInstitute/ecephys_spike_sorting/blob/master/ecephys_spike_sorting/modules/mean_waveforms/waveform_metrics.py
+15/04/2020
+"""
```

Comment:
Example comment @TRuikes: 

1. We could allow to upsample the templates
2. And to compute features on all channels (or a subset of maximum channels)

Reply:
1. how would you interpolate the signal?
2. `waveform` in all functions can be a 2D matrix, I'd handle what exact channels to pass one lever higher